Syntax error in tests/interp_mod.f90

I am getting this syntax error when I compile on wcoss2 TDS (acorn)
```
[ 91%] Building Fortran object tests/CMakeFiles/test_library.dir/interp_mod.f90.o
cd /lfs/h2/emc/ptmp/Dusan.Jovic/simple-ufs/libs/nceplibs/NCEPLIBS-ip/build/tests && /opt/cray/pe/craype/2.7.0/bin/ftn  -I/lfs/h2/emc/ptmp/Dusan.Jovic/simple-ufs/libs/nceplibs/NCEPLIBS-ip/build/src/include_d  -r8 -g -traceback -assume byterecl -O3   -c /lfs/h2/emc/ptmp/Dusan.Jovic/simple-ufs/libs/nceplibs/NCEPLIBS-ip/tests/interp_mod.f90 -o CMakeFiles/test_library.dir/interp_mod.f90.o
/lfs/h2/emc/ptmp/Dusan.Jovic/simple-ufs/libs/nceplibs/NCEPLIBS-ip/tests/interp_mod.f90(347): error #7912: An explicit INTENT(OUT) declaration is expected for this dummy argument.   [GETARG]
    call getarg(i1, grid)
--------------------^
/lfs/h2/emc/ptmp/Dusan.Jovic/simple-ufs/libs/nceplibs/NCEPLIBS-ip/tests/interp_mod.f90(349): error #7912: An explicit INTENT(OUT) declaration is expected for this dummy argument.   [GETARG]
    call getarg(i1, interp_opt)
--------------------^
compilation aborted for /lfs/h2/emc/ptmp/Dusan.Jovic/simple-ufs/libs/nceplibs/NCEPLIBS-ip/tests/interp_mod.f90 (code 1)
make[2]: *** [tests/CMakeFiles/test_library.dir/build.make:76: tests/CMakeFiles/test_library.dir/interp_mod.f90.o] Error 1
make[2]: Leaving directory '/lfs/h2/emc/ptmp/Dusan.Jovic/simple-ufs/libs/nceplibs/NCEPLIBS-ip/build'
make[1]: *** [CMakeFiles/Makefile2:1039: tests/CMakeFiles/test_library.dir/all] Error 2
```

--- a/tests/interp_mod.f90
+++ b/tests/interp_mod.f90
@@ -343,10 +343,6 @@ contains
     integer :: grd218(200)  ! lambert conformal (ncep grid 218) 
     data grd218 /3, 614, 428, 12190, -133459, 8, -95000,  &
          12191, 12191, 0, 64, 25000, 25000, 0, 0, 0, 0, 0, 0, 255, 180*0/
-    i1=1
-    call getarg(i1, grid)
-    i1=2
-    call getarg(i1, interp_opt)
 
     select case (trim(grid))
     case ('3')

